Kitodo/src/main/java/de/sub/goobi/export/dms/ExportDms.java

```diff
@@ -494,9 +494,13 @@ public void imageDownload(Process process, URI userHome, String atsPpnBand, fina
                  * wenn kein Agora-Import, dann den Ordner mit
                  * Benutzerberechtigung neu anlegen
                  */
-                User myUser = (User) Helper.getManagedBeanValue("#{LoginForm.myBenutzer}");
+                User user = Helper.getCurrentUser();
                 try {
-                    fileService.createDirectoryForUser(zielTif, myUser.getLogin());
+                    if (user != null) {
+                        fileService.createDirectoryForUser(zielTif, user.getLogin());
+                    } else {
+                        throw new IOException("noLoggedUser");
```

Comment:
An exception messages must not be translatable and they are not translated until now.